otsd

```diff
@@ -104,8 +101,12 @@ calendar_path = os.path.expanduser(args.calendar_path)
 calendar = otsserver.calendar.Calendar(calendar_path)
 aggregator = otsserver.calendar.Aggregator(calendar)
 
+proxy = bitcoin.rpc.Proxy()
+info = proxy.getinfo()
+relayfee = float(info['relayfee'])
+
 stamper = otsserver.stamper.Stamper(calendar,
-                                    args.btc_min_relay_feerate / 1000 * bitcoin.core.COIN,
+                                    relayfee / 1000 * bitcoin.core.COIN,
```

Comment:
But again, if we happen to start otsd when the node has a particularly high min relay fee we'll be overpaying; remember that the minimum relay fee is set dynamically based on the contents of the mempool.

This is really a minimum relay fee _bump_, and the only reason it can't be just 1 is that'd take quite awhile spinning it's wheels; 1000 is a perfectly good default given how Bitcoin core works right now.